samples/grpc/restaurant-drone-deliveries-service-scala/build.sbt

```diff
@@ -38,7 +38,7 @@ val AkkaDiagnosticsVersion = "2.0.1"
 enablePlugins(AkkaGrpcPlugin)
 
 enablePlugins(JavaAppPackaging, DockerPlugin)
-dockerBaseImage := "docker.io/library/eclipse-temurin:17.0.3_7-jre-jammy"
+dockerBaseImage := "docker.io/library/eclipse-temurin:17.0.8.1_1-jre"
```

Comment:
it will be difficult to maintain if we don't try to keep all of the samples of the same things, change in all places?